src/app/pages/ArticlePage/index.test.tsx

```diff
@@ -52,11 +52,6 @@ jest.mock('../../components/ATIAnalytics');
 jest.mock('#app/legacy/containers/OptimizelyArticleCompleteTracking');
 jest.mock('#app/legacy/containers/OptimizelyPageViewTracking');
 
-jest.mock('#app/hooks/useOptimizelyVariation', () => ({
-  __esModule: true,
-  default: jest.fn(),
-}));
```

Comment:
Think this needs kept actually to fix the failing test

Reply:
I have put it back and everything is passing now